integration-tests/tests/docload/docload.spec.js

```diff
@@ -48,16 +52,14 @@ module.exports = {
     await browser.timesMakeSense(docFetch.annotations, 'requestStart', 'responseStart');
     await browser.timesMakeSense(docFetch.annotations, 'responseStart', 'responseEnd');
     await browser.timesMakeSense(docFetch.annotations, 'fetchStart', 'responseEnd');
-    if (browser.options.desiredCapabilities.browserName !== 'Safari') {
+    if (browserName !== 'Safari') {
```

Comment:
Can we start extracting browser conditions to helpers? Or at least compare after lower-casing browser names?

Reply:
On second thought, this is repeated so many times that it should be extracted to a helper.